proto/ds/dsdata.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Let's remove this change from your PR since you did not modify the `dsdata.proto` file. This change is likely due to protoc is not run when new changes were added to `dsdata.proto`. However, let's leave this task to PRs that are making use of those objects.